src/gpl/src/mbff.cpp

```diff
@@ -137,6 +137,11 @@ float MBFF::GetDist(const Point& a, const Point& b)
   return (abs(a.x - b.x) + abs(a.y - b.y));
 }
 
+float MBFF::GetDistAR(const Point& a, const Point& b, const float AR)
```

Comment:
What is AR?

Reply:
aspect ratio-based distance for initial mbff generation